pwa/src/templates/templateParts/filters/FiltersTemplate.tsx

```diff
@@ -91,6 +105,20 @@ export const FiltersTemplate: React.FC<FiltersTemplateProps> = ({ isLoading }) =
     navigate(`/${filtersToUrlQueryParams(filters)}`);
   }, [filters]);
 
+  const getItems = useFilterCount().getCategoryCount();
+
+  React.useEffect(() => {
+    if (!getItems.isSuccess) return;
+
+    const categoriesWithData = getItems.data.Categorie.map((test: any) => {
+      return TEMP_PUBLICATION_TYPES?.find((option) => {
+        return option.value === test._id.toLowerCase();
+      });
```

Comment:
Do we actually still need these hardcoded `TEMP` values? I think the gateway returns enough for us to remove this temp data.